tests/tests_default.yml

```diff
@@ -1,5 +1,19 @@
 - name: Ensure that the rule runs with default parameters
   hosts: all
-
-  roles:
-    - linux-system-roles.kdump
+  tasks:
+    - name: >-
+        The role requires reboot only on specific systems. Hence running the
+        role in a rescue block to catch when it fails with reboot.
+      block:
+        - name: Run the role. If reboot is not required - the play succeeds.
+          include_role:
+            name: linux-system-roles.kdump
+          when: not kdump_reboot_required | d(false)
```

Comment:
`kdump_reboot_required` will not be set here - unless I am mistaken, and the `when` clause value is evaluated after the role is included?

Reply:
Yes, this was redundant, sorry. Removed.